.github/workflows/test.yml

```diff
@@ -0,0 +1,187 @@
+name: "Test app initialization"
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+        - main
+        - staging
+
+env:
+    RPC_URL_1: ${{ secrets.RPC_URL_1 }}
+    RPC_URL_11155111: ${{ secrets.RPC_URL_11155111 }}
+    RPC_URL_8453: ${{ secrets.RPC_URL_8453 }}
+    RPC_URL_84532: ${{ secrets.RPC_URL_84532 }}
```

Comment:
Why do we have both env var formats here?

Reply:
mainly to handle the transition from the old to the new without any hiccups. this is just in automation and the end user won't see it

Reply:
OK -- to check we're aligned, the policy here will be to export the full set of env variables in each repo, but prefer the `RPC_URL_*` ones in newly referenced cases?

Reply:
Made a tracking ticket here: https://linear.app/intrinsictech/issue/INT-1384/[overall]-migrate-env-variables-from-provider-uri-to-rpc-url-

Reply:
correct, we will want to export the full set for now just to ensure that there's full coverage. after the migration, we can remove the PROVIDER_URI_* vars as they will be unused.